lib/usecase/base/use_case_result.dart

```diff
@@ -20,4 +20,10 @@ class Failed<T> extends Result<T> {
   final UseCaseException exception;
 
   Failed(this.exception) : super._();
+
+  String? getErrorMessage() {
```

Comment:
I think we don't need to handle `null`
```suggestion
  String getErrorMessage() {
```